/*
  Vehicle Data

  This API section holds messages related to OTA Vehicle Data between device and backend.

  The version of the OpenAPI document: 3

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.vehicledata.v3;

import public "device_to_backend_message_meta.proto";
import public "status.proto";

message MeasurementStatusEvent {

  DeviceToBackendMessageMeta meta = 3347973;

  Status status = 355610639;

  // Device specific error code
  string errorCode = 329035797;

  // A descriptive error message
  string errorMessage = 129494241;

}
